### ISSUE TITLE
Prevent frequent crashes by using The sun moon plugin. (O3DS)

### ISSUE THREAD

**Issue by Naturally-Delicious** [imaboy321/Ledybot#24]
(Sorry if this is common knowledge)
So, if you install this plugin: https://gbatemp.net/threads/release-pokémon-sumocheatmenu-supports-o3ds.455365/ and enable NTR Debug Connection, the bot crashing lowers significantly. (at least for me)

**Comment by SteveCookTU** [comment]
This is because the default O3DS NTR does not fully support debugging mode which ledybot uses. Good to know this helps!

**Comment by gr33nd3v1l** [comment]
@Naturally-Delicious have you had problems when starting up the games? (the whited out "closing app..." screen) and do you have any tips how to avoid it\make it start more often?

**Comment by Naturally-Delicious** [comment]
Yes. When starting up the game, rapidly press A untill you youre in game and can see your trainer. It freezes up when starting because there are so many cheats to load.

**Comment by gr33nd3v1l** [comment]
Yeah.. it either hangs on that whiteout screen for me, or loads normally =|

Gamble every time

**Comment by SteveCookTU** [comment]
Interesting. If possible send me the latest batch with no .bak files so I can test it and attempt to replicate the issue.

**Comment by SteveCookTU** [comment]
Ignore that last post, wrong issue haha.
